### ISSUE TITLE
Bug in reverting -of labels

### ISSUE THREAD

**Issue by RikVN** [freesunshine0316/sembleu#3]
When checking if something is a -of relation, you check if the label ends with "-of" (line 251 of amr_graph.py). But you also do this for concepts, so when my model predicted "k / kind-of", it interpreted it as a -of relation, and the script errorred.

An easy fix is to add:

and e.label.startswith(":"):

to line 251 in amr_graph.py. Thanks for looking into it!

**Comment by freesunshine0316** [comment]
Hi @RikVN Thanks for pointing that out, we have fixed it as recommended.

**Comment by danielgildea** [comment]
It looks like there is no ":" in the internal representation, so this change has the effect of preventing any edges from being reversed.  I'm not able to reproduce the original problem with "k / kind-of".  Can you send an example?

**Comment by RikVN** [comment]
I've tried to reproduce the issue in the past half hour, with different output files and different commits of the repo (including my own version from last year). I think there were two things that went wrong: (1) I used a commit before your commit of August 23rd, 2019 and (2) I misidentified the problem. 

I can get the (old) SemBleu to error with AMRs such as this one, which occurred in my output (Seq2seq models can do weird things):

(n / name :op1 ":example-of")

So I don't really remember this, but I assume I saw example-of in the terminal when debugging, misidentified what was going on and thought it that it was an easy fix to check for the colon. This "worked" because now the whole reverse step was skipped. But if I use a version after the commit of 23 August 2019, I don't even get the error, so the fix wouldn't have been necessary anyway.

To sum up it looks I introduced a bug in the system rather than fix one, so I apologize for that. Hope it didn't have an influence on published papers & scores.

**Comment by danielgildea** [comment]
No problem, thanks for looking into it.
